Gruntfile.js

```diff
@@ -12,7 +12,6 @@ module.exports = function(grunt) {
             lib          : 'lib',
             build        : 'build',
             dependencies : 'dependencies',
-            release      : 'release',
```

Comment:
Previously, Babel would output into `build` and then that file would be wrapped in an AMD definition and put in `release`. Now, webpack builds a UMD library, no intermediate step required.